ui/component/commentsList/view.jsx

```diff
@@ -6,10 +6,25 @@ type Props = {
   comments: Array<any>,
   fetchComments: string => void,
   uri: string,
+  claimIsMine: boolean,
+  myChannels: ?Array<ChannelClaim>,
 };
 
 function CommentList(props: Props) {
-  const { fetchComments, uri, comments } = props;
+  const { fetchComments, uri, comments, claimIsMine, myChannels } = props;
+
+  // todo: implement comment_list --mine in SDK so redux can grab with selectCommentIsMine
+  const isMyComment = (channelId: string) => {
```

Comment:
This was causing an error when I refreshed on a page with comments
```
"Cannot read property 'length' of undefined
    at isMyComment"
```

Reply:
Was using `myChannels !== null` instead of `myChannels != null`